### DISCUSSION TITLE
Visible line showing scanning direction

### DISCUSSION THREAD

**Discussion by kuhlaid** [mebjas/html5-qrcode#262] [Ideas]
When using a barcode scanner (a device that ONLY scans barcodes), the user usually has a visual red line that illuminates the barcode. This helps the user know if they have the scanner aligned with the barcode correctly. This app (https://serratus.github.io/quaggaJS/examples/live_w_locator.html)[https://serratus.github.io/quaggaJS/examples/live_w_locator.html] shows a 'red line' while scanning and it is a useful visual feature as it mimics a barcode scanner.
![Screenshot_20210801-161634](https://user-images.githubusercontent.com/41551429/127784365-c77fb052-6bcf-4868-b4e9-bdad879bafe2.png)

**Comment by ROBERT-MCDOWELL** [comment]
yup, seems to be a good new feature to add for horizontal barcode indeed.
